src/components/PaginationItem/PaginationItem.tsx

```diff
@@ -1,8 +1,18 @@
 import classNames from "classnames";
 import PropTypes from "prop-types";
-import React from "react";
+import React, { MouseEventHandler } from "react";
```

Comment:
Should this be `import type { MouseEventHandler } from "react";` like above?

Reply:
It should, thanks @huwshimi!